pkgs/by-name/as/asfa/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  gawk,
+  help2man,
+  libssh2,
+  openssl,
+  pkg-config,
+  ...
+}: let
+  version = "0.10.0";
+in
+  rustPlatform.buildRustPackage {
+    pname = "asfa";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "obreitwi";
+      repo = "asfa";
+      rev = "v${version}";
+      hash = "sha256-MnhnwtZmPFhOuiqNiaxJnPu88JOdlpvyVy0YGphblBc=";
+    };
+
+    cargoHash = "sha256-/bRBP/NzcNOXl/nANeOYouUAo3NNbtbV9fxIJrNajYQ=";
+
+    buildInputs = [
+      openssl
+    ];
+
+    nativeBuildInputs = [
+      gawk
+      help2man
+      pkg-config
+    ];
+
+    doCheck = false;
+
+    postInstall = ''
+      mkdir -p "$out/man/man1"
+      help2man -o "$out/man/man1/asfa.1" "$out/bin/asfa"
+
+      # Generate info about all subcommands except for 'help' (which leads to error)
+      "$out/bin/asfa" --help | awk 'enabled && $1 != "help" { print $1 } /^SUBCOMMANDS:$/ { enabled=1 }' \
+          | while read -r cmd; do
+          help2man \
+              "--version-string=${version}" \
+              -o "$out/man/man1/asfa-''${cmd}.1" \
+              "$out/bin/asfa ''${cmd}"
```

You could use `$var` instead of `''${var}`
This might be better, since less escaping is needed

```suggestion
              -o "$out/man/man1/asfa-$cmd.1" \
              "$out/bin/asfa $cmd"
```

---

Good point, adapted.